src/PrestaShopBundle/Form/Admin/Type/DatePickerType.php

```diff
@@ -45,6 +57,11 @@ public function getParent()
         return TextType::class;
     }
 
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->addViewTransformer($this->arabicToLatinDigitDataTransformer);
```

Comment:
One question, since the transformer converts arabic to latin, back and forth Doesn't it risk to transform latin number into arabic ones even when not needed? How is it able to detect when it must be used or not?

Reply:
🤯 You are right! I'll have to find a way 😅

Reply:
@atomiix You didn't change anything about this issue, so how do you transform in one way only?